### ISSUE TITLE
Scope black / charcoal / coal / charbon, etc.

### ISSUE THREAD

**Issue by thuchacz** [cu-mkp/m-k-manuscript-data#1112]
NJR to scope: blacks / charbon / charcoal / coal

- charbon translate as “carbon black”?  It is not common charcoal, i.e., wood,, but general vegetable blacks - glossary 
- CHECK: NB: coal - definition at this time - sea coal [check our translation on this, and glossary/comments?]
- Charbon de pierre (syn. charbon de terre) 63v pit coal; 100r ; 
- 141 instances of “*coal*” in TL
- Charcoal / charbon = carbon black?
- Do we want editorial comment? In glossary?

**Comment by njr2128** [comment]
Search in oxygen:
![image](https://user-images.githubusercontent.com/14779727/64978585-dc4f2300-d883-11e9-91e0-2b25070b5fd6.png)

Working file in GD: [coal_tl](https://docs.google.com/spreadsheets/d/1mp4rJAR5kMzkytNopRQxMsixgRHJ7Be_T9NH5DTVUW0/edit?usp=sharing)

**Comment by njr2128** [comment]
@ps2270, all instances of "*coal*" (141 times) have been gathered in the GD doc, [coal_tl ](https://docs.google.com/spreadsheets/d/1mp4rJAR5kMzkytNopRQxMsixgRHJ7Be_T9NH5DTVUW0/edit?usp=sharing) and is ready for your review.

For each occurrence, I have listed the word used in the tl (charcoal or coal) and matched it to the word used in tcn (which are all variations of charbon). 
I have then categorized the occurrences by "context," e.g., used as a pigment, used to mean a charcoal fire, used for heating, or as residue in molds (among others).
Finally, I have put very few notes about inconsistent uses that I can see just from this survey.

In short, we have inconsistently used "coal" in heat/fire contexts a few times, where elsewhere in the translation we have used "charcoal" for very similar sentences/wording.
Another inconsistency I found was the use of both "lit charcoal" and "glowing charcoal" for the french "charbons allumés" (lit was much more frequent than glowing).

**Comment by njr2128** [comment]
For some information about "sea coal," Marika Spring et al's _Black Earths': A Study of Unusual Black and Dark Grey Pigments used by Artists in the Sixteenth Century_ provides a nice summary:

- References to the use of coal as a black pigment in Northern seventeenth-century painting treatises
are well known. Theodore de Mayerne, for example, mentions charbon de pierre and charbon de terre d'Escosse , both common coal, and it also appears under the names of sea coal and smythe coal (forge coal) in several English treatises of the period.27 The earliest reference seems to be in The Arte of Limning by Nicolas Hilliard dating from the beginning of the seventeenth century.28 Edward Norgate stated that 'sea cole makes a Red Blacke', which is consistent with the warm black appearance of the coal that has been identified in paintings; in a second version of his treatise he even lists it among the brown pigments.29 Several writers recommend its use for shadows, particularly in flesh paint.30 The sea coal that was collected from shallow deposits on the coast was probably a low-grade coal, which would be likely to have a high sulphur content; it was therefore probably similar to the coal pigment identified in paintings in this study.31 As noted earlier, none of the names of black pigments referred to in sixteenth-century Italian painting treatises can be specifically identified with common coal. Biringuccio confirms that it was available in Italy during this period, however, describing its use as a fuel in his Pirotechnia

- 27 Van de Graaf, cited in note 3, pp. 148 and 151. Harley, cited in note 3, refers to five English seventeenth-century treatises that mention coal as a pigment, see p. 157. See also C. L. Eastlake, Methods and Materials of Painting of the Great Schools and Masters, New York i960, Vol. 1, p. 467, for references to coal in van Mander and MS Harley 6376. A Spanish seventeenth-century treatise on painting by Hidalgo mentions carbon de piedra molida, which is probably coal. See Veliz, cited in note 8, p. 136 and n. 9. 

- 28 Hilliard, cited in note 4, p. 99. 

- 29 Eastlake, cited in note 27, p. 467. The quote is from MS 6376, which is a variant of the first version of Norgate's treatise (for the history of this manuscript see J, Kirby, 'The Painter's Trade in the Seventeenth Century: Theory and Practice', National Gallery Technical Bulletin, 20, 3:999, P* 4Z> n* 38). Seacole is listed with the brown pigments in the second version of Edward Norgate's Miniatura or the Art of Limning, eds J. M. Muller and J. Murrell, New Haven and London 1997, p. 59. However, in a variant list from a copy of Norgate's first treatise (National Library of Wales 21753 91) lt 1S listed with the black pigments, see eds Muller and Murrell, cited above, p. 254. 

- 30 Hilliard, cited in note 4, p. 99, when discussing how to depict pearls, recommends applying 'a small shadow of seacoal undermost of all'. De Mayerne in van de Graaf, cited in note 3, p. 148, suggests coal for rendering the shadows of flesh, as does Henrie Peacham: H. Peacham, Graphice or The Most Ancient and Excellent Art of Drawing and Limning, London 1612, p. 95. 

- 31 Hilliard, cited in note 4, indicates that sea coal had a high sulphur content, 'the culers them sellues may not endure some ayers, especially in the sulfirous ay re of seacole', p. 74. Brimblecombe, cited in note 21, discusses the foul, sulphurous fumes produced by burning medieval sea coal, pp. 7-9

**Comment by ps2270** [comment]
@njr2128  is going to pull all charbons vs charbon and PHS will continue working on this.

**Comment by njr2128** [comment]
@ps2270, the [coal spreadsheet](https://docs.google.com/spreadsheets/d/1mp4rJAR5kMzkytNopRQxMsixgRHJ7Be_T9NH5DTVUW0/edit?usp=sharing) has been updated with a new sheet for charbon in the french.

To pull the results, a search in oxygen was made for "charb" in the tcn (to catch for any expansions, etc).

**Comment by ps2270** [comment]
I've gone through tcn and tl, and added charbons to the spreadsheet for the tl.  He uses "charbons" rather than "charbon" 41 times, and most of these should be "coals" instead of "charcoals" (which I have not found to be an English word) b/c he is talking about individual coals--e.g., make a ring of coals, or the big coals, etc..  There are a few fœu de <m>charbons</m> and "fœu de <m>charbon</m>," and I have a question whether we want to to distinguish between a charcoal fire, and a fire of coals.  They could mean different things.  @Pantagrueliste @TillmannTaape @thuchacz , what do you think?

**Comment by njr2128** [comment]
DECISION:
all "charbon" --> charcoal
all "charbons" --> charcoals

**Comment by njr2128** [comment]
All changes recorded in 
## [coal spreadsheet](https://docs.google.com/spreadsheets/d/1mp4rJAR5kMzkytNopRQxMsixgRHJ7Be_T9NH5DTVUW0/edit?usp=sharing)

For any changes where more than just adding an "s" to charcoal was necessary, notes have been left showing the intervention.
The main change of this nature was changing:
ORIGINAL: "charcoal fire" to
NEW: "fire of charcoals"
for all instances of _feu de charbons_

@ps2270, please review and close issue if everything is ok with you

**Comment by ps2270** [comment]
I've responded in the spreadsheet.  Please do not translate "feu de charbons" as "fire of charcoals", but leave it as "charcoal fire", as CAG explained yesterday.
The others are ok, except..."if there are charcoals inside..." This sounds incomprehensible to me. If we don't translate as "cinders," which is what it clearly means, then we should leave it as "charcoal" singular.
Please make these changes, then close the issue.
thanks.

**Comment by njr2128** [comment]
I have recorded the last changes next to your comments in the spreadsheet.
I did not use "cinders" but instead changed back to the singular "charcoal" as suggested.
